ansible/roles/machine/setup/tasks/install_packages.yml

```diff
@@ -45,6 +45,42 @@
     - xorg-x11-server-Xvfb
     - firefox
     - python3-paramiko
+    - firewalld
+
+# Configure firewalld (make sure that ssh is enabled)
+# This is using the command line tool till python2/3 firewalld import issues
+# are really solved for the ansible firewalld module
+#
+# This can be used later on:
+#   firewalld:
+#     service : "{{ item }}"
+#     permanent: true
+#     state: enabled
+#   with_items:
+#     - ssh
+#
+- name: Configure firewalld
```

Comment:
FirewallD is not running after its installation:

```bash
TASK [machine/setup : Configure firewalld] ***************************************************************************************************
fatal: [192.168.121.229]: FAILED! => {"changed": true, "cmd": ["firewall-cmd", "--permanent", "--add-service=ssh"], "delta": "0:00:00.311440", "end": "2018-11-26 02:32:24.087140", "msg": "non-zero return code", "rc": 252, "start": "2018-11-26 02:32:23.775700", "stderr": "FirewallD is not running", "stderr_lines": ["FirewallD is not running"], "stdout": "", "stdout_lines": []}
```

Reply:
Using `firewall-offline-cmd` instead of `firewall-cmd --permanent` is fixing this